Contributing.md

```diff
@@ -0,0 +1,73 @@
+﻿# Contributing guidelines
+
+Welcome to the **good-first-issues** project!. All the new contributors can use this ready reference to kick start their contribution to this project.
+Feel free to ask your queries anytime and remember, no contribution is too small :)
+
+
+# Lets Start!
+Follow the steps below and feel free to read from the resources given at the end, in case something is not clear.
+
+## STEP 1:
```

Comment:
`Step` is wholly capitalized on some parts, and not on others. Follow consistency in headings and titles